src/app/login/LoginForm.tsx

```diff
@@ -81,7 +81,7 @@ export default function LoginForm() {
       borderRadius={'0 0 80% 0'}
       alignItems={'center'}
       justifyContent={'center'}
-      padding={{ xs: '4em 4em', lg: '4em 4em' }}
+      padding={{ xs: '4em 0em', lg: '4em 4em' }}
     >
```

Comment:
This is nice addition since it will make the login more responsive in mobile, also can you please add loading state for loading and register?

Reply:
Alright done! I have updated my original comment!